resources/schema/spidacalc/results/item.schema

```diff
@@ -29,7 +29,12 @@
       "$ref": "../../general/measurable.schema"
     },
     "reaction": {
-      "$ref": "../../general/point_load.schema"
```

Comment:
@tgsoverly Is this ok? Had to change it to this because of: https://github.com/spidasoftware/calc/blob/master/engine/src/main/groovy/com/spidasoftware/engine/fem/results/factories/LoadingResultsJsonFactory.groovy#L380-L385

It wasn't validating against the schema.